plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -288,6 +288,14 @@ public void testNullPredicate()
 
```

Comment:
Please change commit message: "Added a test for null or value clause in Elasticsearch connector"

Reply:
Done.